src/test/scala/edu/utah/cs/simba/SimbaSuite.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 by Simba Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.utah.cs.simba
+
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.sql.SQLContext
+import org.scalatest.FunSuite
+
+import scala.collection.mutable.ListBuffer
+
+/**
+  * Created by gefei on 2016/11/14.
+  */
+class SimbaSuite extends FunSuite{
```

Comment:
SimbaSuite does not work here, gives me a Serialization Problem, so I tried to use example package to test.